elastic/beats
# Add host inventory metrics to googlecloud compute metricset

## What does this PR do?



## Why is it important?

This PR is to add proposed host common fields into `compute` metricset:
- host.id
- host.name
- host.cpu.pct
- host.network.in.bytes
- host.network.in.packets
- host.network.out.bytes
- host.network.out.packets
- host.disk.read.bytes
- host.disk.write.bytes

## Checklist




## How to test this PR locally

1. Start Metricbeat `googlecloud` module: `./metricbeat modules enable googlecloud`
2. Edit `modules.d/googlecloud.yml` to only include `compute` metricset:
```
- module: googlecloud
  metricsets:
    - compute
  region: "us-east1"
  project_id: elastic-observability
  credentials_file_path: "/Users/kaiyansheng/Downloads/elastic-observability-d17781618202.json"
  exclude_labels: false
  period: 300s
```
3.Change add_host_metadata processor config in metricbeat.yml file:
```
processors:
  - add_host_metadata:
      replace_fields: false
```
4. Start metricbeat
5. You should see metrics from `compute` metricset and includes fields listed above.
